food-truck.eve.md

```diff
@@ -436,12 +539,14 @@ search @browser
   [#truck-name value]
```

Comment:
This flow could use some UX love. De-focusing the input and hitting esc should do something here. Also, what does the edit button do here?